mirage/apt/apt_inputs.py

```diff
@@ -338,6 +340,39 @@ def create_input_table(self, verbose=False):
         ascii.write(Table(self.exposure_tab), self.output_csv, format='csv', overwrite=True)
         print('csv exposure list written to {}'.format(self.output_csv))
 
+
+    def check_aperture_override(self):
+        instruments = self.exposure_tab['Instrument']
+        apertures = self.exposure_tab['aperture']
+
+        aperture_key = {'nircam': 'NRC',
```

Comment:
Since this dictionary is of general use (I think there are similar ones in other parts of mirage) I suggest to move it to a generic place (`utils/constants.py`?) so that it can be reused when necessary. 
(The same applies to other general definitions, see the top section of `catalog_seed_image.py` for instance but that's maybe for a separate cleanup PR.)

Reply:
Good idea - done for this case. (I saw that you added a `utils/constants.py` file in another PR, so there might end up being a merge conflict down the line.)